index.bs

```diff
@@ -0,0 +1,139 @@
+<pre class='metadata'>
+Title: Audio Session
+Shortname: audio-session
+Level: None
+Status: w3c/ED
+Group: mediawg
+TR: https://www.w3.org/TR/audio-session/
+Repository: w3c/audio-session
+URL: https://w3c.github.io/audio-session/
+Editor: Youenn Fablet, Apple https://www.apple.com/, youenn@apple.com, w3cid 96458
+Editor: Alastor Wu, Mozilla https://www.mozilla.org, alwu@mozilla.com, w3cid 92198
+Abstract: This API defines an API surface for manipulating the bits on
+Markup Shorthands: css no, markdown yes
+</pre>
+<pre class=link-defaults>
+spec:webidl; type:dfn; text:resolve
+</pre>
+
+# Introduction # {#introduction}
+
+People consume a lot of media (audio/video) and the Web is one of the primary means of consuming this type of content.
+However, media on the web does not integrate well with the platform.
+The Audio Session API helps to close the gap with platforms that have audio session/audio focus such as Android and iOS.
+This API will help by improving the audio-mixing of websites with native apps, so they can play on top of each other, or play exclusively.
+
+Additionally, on some platforms the user agent will automatically manage the audio session for the site
+based on whether media elements are playing or not and which APIs are used for playing audio.
+In some cases this may not match user expectations, this API provides overrides to authors.
+
+# The {{AudioSession}} interface # {#audio-session}
+
+By convention, there are several `audio session types` for different purposes:
+
+ * Playback (`playback`) audio, which is used for video or music playback, podcasts, etc. They should not mix with other playback audio. (Maybe) they should pause all other audio indefinitely.
+ * Transient (`transient`) audio, such as a notification ping. They usually should play on top of playback audio (and maybe also "duck" persistent audio).
+ * Transient solo (`transient-solo`) audio, such as driving directions. They should pause/mute all other audio and play exclusively. When a transient-solo audio ended, it should resume the paused/muted audio.
+ * Ambient (`ambient`) audio, which is mixable with other types of audio. This is useful in some special cases such as when the user wants to mix audios from multiple pages.
+ * Play and record (`play-and-record`) audio, which is used for recording audio. This is useful in cases microphone is being used or in video conferencing applications.
+ * Auto (`auto`) lets the User Agent choose the best audio session type according the use of audio by the web page. This is the type of the default AudioSession.
+
+The AudioSession is the main interface for this API. It can have the following states:
+
+ * active: the AudioSession is playing sound.
+ * interrupted: the AudioSession is not playing sound, but can resume when it will get uninterrupted.
+ * inactive: the AudioSession is not playing sound.
```

Comment:
```suggestion
 * {{AudioSessionState/active}}: the {{AudioSession}} is playing sound.
 * {{AudioSessionState/interrupted}}: the {{AudioSession}} is not playing sound, but can resume when it will get uninterrupted.
 * {{AudioSessionState/inactive}}: the {{AudioSession}} is not playing sound.
```